src/tinycthread/tinycthread.h

```diff
@@ -28,15 +29,6 @@ freely, subject to the following restrictions:
 extern "C" {
 #endif
 
-// jcheng 2017-11-03: _XOPEN_SOURCE 600 is necessary to prevent Solaris headers
-// from complaining about the combination of C99 and _XOPEN_SOURCE <= 500. The
-// error message starts with:
-// "Compiler or options invalid for pre-UNIX 03 X/Open applications"
```

Comment:
Is it OK that we're losing this?

Reply:
I think we want to keep this and maybe a few other things -- I haven't yet reviewed the changes we made to tinycthread that we may want to preserve in the new version.